### ISSUE TITLE
UPM not correctly calculated in vertical-metrics.py

### ISSUE THREAD

**Issue by davelab6** [googlefonts/fontbakery#504]
As per https://github.com/rosettatype/Eczar/issues/9#issuecomment-71918155 there's a bug in the UPM calculation of the vertical-metrics fixer

**Comment by vitalyvolkov** [comment]
this https://github.com/googlefonts/fontbakery/commit/3a72aa8ea343ecb2751a13d825fa9878ce5b796c should close this issue

but i'm not sure that it is correct. in which case upm can be changed? i think we should have another fixer for upm. what do you think?

**Comment by davelab6** [comment]
A Upm fixer would likely need to scale all glyphs and GPOS positions...
It's a big job. Better to do that fixing by hand.

**Comment by davelab6** [comment]
Yikes, this is a real problem. 

When I processed Tillana and Poppins, the Tillana Bold and ExtraBold's hhea.unitsPerEm got changed from 1000 to 1158, and Poppins Bold's got changed from 1000 to 1115. 

I fixed these with TTX by hand - https://code.google.com/p/googlefontdirectory/source/detail?r=069ffdd1685896d9a9ef2a8566039f6cef60e626 https://code.google.com/p/googlefontdirectory/source/detail?r=fbdb5f71d0113ad37f12d3708f1ccb0383961b47 - but this should not happen. 

The UPM value should never be changed.
